src/PhpRenderer.php

```diff
@@ -191,6 +190,18 @@ public function fetchTemplate(string $template, array $data = []): string
         return $output;
     }
 
+    /**
+     * Returns true is template exists, false if not
+     *
+     * @param $template
+     *
+     * @return bool
+     */
+    public function hasTemplate(string $template) : bool
```

Comment:
Can we rename this to `templateExists()`? I think it would be more clear

Reply:
I don't strongly object, although I think the `has`, `set`, `get` paradigm is clear and very common.

Are we sure about this?

Reply:
`hasTemplate` to me implies that the Renderer holds internal state and has that template. I think `templateExists` is less ambiguous. Clearly, this is subjective and I'm not saying you're wrong and I'm right. I'd rather use the `templateExists` method since that's exactly what the method is doing. We should also add `is_readable()` to this check.

Reply:
using exists would more accurate reflect that the template can be rendered and would mimic `file_exists()` style in the language runtime itself.

Reply:
I'll concede. Updated.